### ISSUE TITLE
flux full training on multi gpu validation is block with deepspeed zero3 

### ISSUE THREAD

**Issue by YLongJin** [bghira/SimpleTuner#965]
logs = {
                    "step_loss": loss.detach().item(),
                    "lr": float(self.lr),
                }
                if "mean_cfg" in wandb_logs:
                    logs["mean_cfg"] = wandb_logs["mean_cfg"]

                progress_bar.set_postfix(**logs)
                self.mark_optimizer_eval()
                self.accelerator.wait_for_everyone()
                print("run_validations 3")
             
                self.validation.run_validations(
                    validation_type="intermediary", step=step
                )
                self.mark_optimizer_train()

in this line：
 self.validation.run_validations(
                    validation_type="intermediary", step=step
                )

flux full training on multi gpu validation is block with deepspeed zero3
